NitroxClient/MonoBehaviours/Gui/HUD/RemotePlayerVitals.cs

```diff
@@ -132,6 +136,22 @@ private void CreatePlayerNameText()
             playerNameText.layer = em.gameObject.layer;
             // em does not have a parent anymore on latest stable, so the stats position is incorrect.
             playerNameText.transform.parent = em.transform;//.parent.transform;
+            GameObject signalBase = Instantiate(Resources.Load("VFX/xSignal")) as GameObject;
```

Comment:
We already create and assign a signal to a player, can that be reused? (preferably)

Reply:
I looked into re-using the signal, but I couldn't off the top of my head figure out a way to get the reference from RemotePlayer into RemotePlayerVitals, unless we can use statics?

Reply:
We have a `PlayerVitalsManager` that returns an existing or new `RemotePlayerVitals` component for a given player. You can use the methods in `RemotePlayer`, but it's up to you to refactor some pieces such that you can access it 😉 

Also, consider where you want to instantiate the label, and how it's gonna be accessed by the other class.